google-cloud-logging/src/main/java/com/google/cloud/logging/CmekSettings.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.logging;
+
+import com.google.common.base.Function;
+import com.google.common.base.MoreObjects;
+import java.util.Objects;
+
+/**
+ * CmekSettings Describes the customer-managed encryption key (CMEK) settings associated with a
+ * project, folder, organization, billing account, or flexible resource.
+ *
+ * <p>Note: CMEK for the Logs Router can currently only be configured for GCP organizations. Once
+ * configured, it applies to all projects and folders in the GCP organization.
+ *
+ * @see <a href="https://cloud.google.com/logging/docs/routing/managed-encryption">Enabling CMEK for
+ *     Logs Router</a> for more information.
+ */
+public class CmekSettings {
+
+  static final Function<com.google.logging.v2.CmekSettings, CmekSettings> FROM_PB_FUNCTION =
+      new Function<com.google.logging.v2.CmekSettings, CmekSettings>() {
+        @Override
+        public CmekSettings apply(com.google.logging.v2.CmekSettings cmekSettingsPb) {
+          return cmekSettingsPb != null ? CmekSettings.fromPb(cmekSettingsPb) : null;
+        }
+      };
+
+  static final Function<CmekSettings, com.google.logging.v2.CmekSettings> TO_PB_FUNCTION =
+      new Function<CmekSettings, com.google.logging.v2.CmekSettings>() {
+        @Override
+        public com.google.logging.v2.CmekSettings apply(CmekSettings cmekSettings) {
+          return cmekSettings != null ? cmekSettings.toPb() : null;
+        }
+      };
+
+  private String name;
+  private String kmsKeyName;
+  private String serviceAccountId;
+
+  /** A builder for {@code CmekSettings} objects. */
+  static class Builder {
+    private String name;
```

Comment:
Builder and its methods must be public, otherwise you export non public API via public method toBuilder()